### PR TITLE
Added settings for elasticsearch.

### PR DESCRIPTION
**Pull request by dsjen**
@dan-f -- this isn't one of the more exciting PRs.

I've added configurations for setting up elastic search.  When it comes time to implement, we can do something like:

```
from elasticsearch_dsl import connections, Index, DocType, String

connections.connections.create_connection(hosts=[settings.ELASTIC_SEARCH_LEARNERS_HOST)
roster = Index(settings.ELASTIC_SEARCH_LEARNERS_INDEX)

# this decorator sets up the index and we could move that step into Meta if this is overkill
@roster.doc_type
class RosterEntry(DocType):
     username = String(index='not_analyzed')
     class Meta:
          doc_type = 'roster_entry'  # this was a key part

# execute search
RosterEntry.search().execute()
```

### CODE REVIEW

File: analyticsdataserver/settings/base.py [hunk 1]
```diff
@@ -49,6 +49,11 @@
     }
 }
 ########## END DATABASE CONFIGURATION
+ELASTIC_SEARCH_LEARNERS_HOST = None
```
Comment (dan-f) [inline analyticsdataserver/settings/base.py]: Nit: I'd remove the first underscore in each name to be: `"ELASTICSEARCH_LEARNERS_HOST"`.

**Comment by dan-f** [comment]
:+1: other than my nitpick.

**Comment by dan-f** [comment]
Actually - could we make this variable be settable from the environment?

File: analyticsdataserver/settings/base.py (hunk 1 above)
Comment (dsjen) [inline analyticsdataserver/settings/base.py]: Ah!  I meant for it not to have an underscore.  Updating.

**Comment by dsjen** [comment]
@dan-f -- what about something like: 
```
ELASTICSEARCH_LEARNERS_HOST = os.environ.get('ELASTICSEARCH_LEARNERS_HOST', None)
```

**Comment by dsjen** [comment]
I updated the config to use environment variables or `None`.

**Pull request merged**